### PR TITLE
Updated Light Theme Shadows to match Storybook

### PR DESCRIPTION
**Pull request by Akatori-Design**
Updated all light-theme shadow tokens to match the [Storybook Spec.](https://metamask.github.io/design-tokens/?path=/docs/shadows-shadows--size)

Description
Added description with "Uses : " examples.

Values:
Updated LG, primary, danger to :
- LG : 0 2 40 0 #0000001A
- primary : 0 2 8 0 #037dd633
- danger : 0 2 8 0 #d73a4966

### CODE REVIEW

**Comment by Akatori-Design** [comment]
renamed `SM-primary` back to `primary`, `SM-danger` back to `danger`

**Comment by Akatori-Design** [comment]
fixed danger shadow size to SM `0 2 8 0 `

File: src/figma/tokens.json [hunk 1]
```diff
@@ -937,71 +937,77 @@
...
-          "blur": 42,
-          "spread": 0
+          "type": "dropShadow"
         },
-        "type": "boxShadow"
+        "type": "boxShadow",
+        "description": "Uses : Primary-Button hover state"
       },
-      "error": {
+      "danger": {
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1000]: @Akatori-Design Should we leave it as error until we do the v2 naming update? Then we can change it all over in one go or do you think it's ok to have some danger and some error? I think we want to be consistent until we covert all of them over?

File: src/figma/tokens.json [hunk 2]
```diff
@@ -937,71 +937,77 @@
       }
     },
     "shadows": {
-      "xs": {
+      "XS": {
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:940]: Can you show me how this looks in Figma in the style panel?

**Review by georgewrmarshall — COMMENTED** [review]
One question about error/danger

**Review by georgewrmarshall — APPROVED** [review]

File: src/figma/tokens.json (hunk 2 above)
Comment (georgewrmarshall) [reply src/figma/tokens.json]: I think we should change it back to lowercase

**Review by georgewrmarshall — APPROVED** [review]

**Pull request merged**